<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::AuthWidget</class>
 <widget class="QWidget" name="Sekura::AuthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sekura authorize</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbDel">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAuth">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Authorized</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Application</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leApp"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cbDefault">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePath"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>/sekura/api/v1.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pbAuth">
       <property name="text">
        <string>Authorize</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="5"/>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>pbDel</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>leApp</tabstop>
  <tabstop>cbDefault</tabstop>
  <tabstop>lePath</tabstop>
  <tabstop>pbAuth</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>cbAuth</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
